helm/trivy/templates/statefulset.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Did you consider using Deployment with emptyDir volumes? I know that you're inspired by aquasecurity/harbor-scanner-trivy, but Harbor adapter is using Trivy in standalone mode. That was the reason to use StatefulSets, but in this case Deployment should work.

Reply:
I understood the comments in a way, that Trivy would pull the database from github.
With the PVC and statefulset I wanted to prevent a pull for the database on every POD restart.

Reply:
@czunker Okey so in case you want to reuse downloaded DB files by configuring the storage class that has the reclaim policy set to `Retain`, having StatefulSet makes sense.  (AFAIK This is not possible with Deployments and the only way is indeed `PersistenceVolumeClaimTemplate`.)

On the other hand we can change the default `podManagementPolicy` of the StatefulSet from `OrderedReady` to `Paraller`, so pods are spun up independently rather than one after the other. There no reason to wait in case of Trivy server. Pods can run and scale up or down independently.

Reply:
I changed the `podManagementPolicy` and added a short note in the Readme.